WordPress/Classes/ViewRelated/Blog/JetpackSecuritySettingsViewController.swift

```diff
@@ -0,0 +1,334 @@
+                                                 value: newValue,
+                                                 success: { [weak self] in
+                                                     self?.reloadViewModel()
+                                                 },
+                                                 failure: { [weak self] (_) in
+                                                     self?.refreshSettingsAfterSavingError()
+                                                 })
+        }
+    }
+
+    fileprivate func matchAccountsUsingEmailChanged() -> (_ newValue: Bool) -> Void {
+        return { [unowned self] newValue in
+            self.settings.jetpackSSOMatchAccountsByEmail = newValue
+            self.service.updateSSOMatchAccountsByEmailForBlog(self.blog,
+                                                              value: newValue,
+                                                              success: {},
+                                                              failure: { [weak self] (_) in
+                                                                  self?.refreshSettingsAfterSavingError()
+                                                              })
+        }
+    }
+
+    fileprivate func requireTwoStepAuthenticationChanged() -> (_ newValue: Bool) -> Void {
+        return { [unowned self] newValue in
+            self.settings.jetpackSSORequireTwoStepAuthentication = newValue
+            self.service.updateSSORequireTwoStepAuthenticationForBlog(self.blog,
+                                                                      value: newValue,
+                                                                      success: {},
+                                                                      failure: { [weak self] (_) in
+                                                                          self?.refreshSettingsAfterSavingError()
+                                                                      })
+        }
+    }
+
+    // MARK: - Footer handler
+
+    func handleLearnMoreTap(_ sender: UITapGestureRecognizer) {
+        guard let url =  URL(string: JetpackSecuritySettingsViewController.learnMoreUrl) else {
+            return
+        }
+        guard let webViewController = WPWebViewController(url: url) else {
+            return
+        }
+
+        if presentingViewController != nil {
+            navigationController?.pushViewController(webViewController, animated: true)
+        } else {
+            let navController = UINavigationController(rootViewController: webViewController)
+            present(navController, animated: true, completion: nil)
+        }
+    }
+
+    // MARK: - Persistance
+
+    fileprivate func refreshSettings() {
+        service.syncJetpackSettingsForBlog(blog,
+                                           success: { [weak self] in
+                                               self?.tableView.reloadData()
```

Comment:
I haven tested yet, but should this be a `reloadViewModel()` instead?

Reply:
Of course it should!